sycl/doc/CompilerAndRuntimeDesign.md

```diff
@@ -510,15 +510,16 @@ down to the NVPTX Back End. All produced bitcode depends on two libraries,
 `libdevice.bc` (provided by the CUDA SDK) and `libspirv-nvptx64--nvidiacl.bc`
 (built by the libclc project).
 
-During the device linking step (device linker box in the
-[Separate Compilation and Linking](#separate-compilation-and-linking)
-illustration), llvm bitcode objects for the CUDA target are linked together
-alongside `libspirv-nvptx64--nvidiacl.bc` and `libdevice.bc`, compiled to PTX
-using the NVPTX backend, and assembled into a cubin using the `ptxas` tool (part
+##### Device code post-link step
+
+During the "PTX target precessing" in the device linking step [Device code post-link step](#device-code-post-link-step), the llvm bitcode objects for the CUDA target are linked together alongside `libspirv-nvptx64--nvidiacl.bc` and `libdevice.bc`, compiled to PTX
```

Comment:
```suggestion
During the "PTX target processing" in the device linking step [Device code post-link step](#device-code-post-link-step), the llvm bitcode objects for the CUDA target are linked together alongside `libspirv-nvptx64--nvidiacl.bc` and `libdevice.bc`, compiled to PTX
```

Reply:
Please format to 80 chars lines.

Reply:
on the picture I can see "libspirv" and "libdevice.bc" passed to "Clang"
1) Looks like "libspirv" should be "libspirv.bc"
2) Looks like "clang" is really PTX back-end - I suggest to replace "Clang" with "PTX backend" then, similarly to the generic DeviceLinkAndWrap terminology

Also does PTX BE accept multiple bitcode files or there is llvm-link before it? If the latter, I'd suggest to reflect that on the picture.

Reply:
I applied your comments on the picture.

It is llvm-link before, but it is clearly shown in the DeviceLinkAndWrap, the picture is a split of the "PTX target processing" box. I added reminder on the input box that it is linked.